### ISSUE TITLE
Print some  PRG • PRG posters!

### ISSUE THREAD

**Issue by tpetricek** [prgprg-org/prgprg-org.github.io#6]
I created a draft poster: https://github.com/prgprg-org/prgprg-org.github.io/blob/gh-pages/materials/prgprg.pdf
If you have any suggestions, please let me know!

I should be able to print some next week and put them on various pinboards around MFF.

**Comment by programLyrique** [comment]
Nice! Did you print them at the end? We might also put some around FIT

**Comment by tpetricek** [comment]
yup! I have one outside of my office and one on the notice board of our department :-)
